Changes on tool readmes (#2388)
Azure/azure-sdk-tools: CONTRIBUTING.md

This is the repository used to host tools and libraries used by Azure SDK team engineering system. 
We allow to create tools in any language suitable for the tools.
For every newly developing tool, create a new folder with brief description on its name. 
Do not add third party tools directly in your folder, use it as dependencies.

## Codeowners

> Add code owner in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/blob/main/.github/CODEOWNERS) with following format:

Reviewer: ```suggestion
Add code owner in [CODEOWNERS](https://github.com/Azure/azure-sdk-tools/blob/main/.github/CODEOWNERS) with following format:
```

## README

> 1. Please add README.md file for every tool to illustrate:

Reviewer 2: The way these numbered items render is probably not what you intend:

![image](https://user-images.githubusercontent.com/1020379/145085617-0bc6b39b-85c7-4d7a-ab28-dcf75a440c79.png)

* The purpose of the tool.
* Prerequisites before use
* How to use, test and maintain the tool locally and remotely. 
* Better to include where the tool is being used.

Example: [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/tools/http-fault-injector/README.md) 

> 2. Please add tool index of the tool in root [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/README.md).

Reviewer 2: In my opinion, "please" sounds a little odd to use in technical documentation. No need for politeness in impersonal and imperative instructions.

Also, I'd direct link to the index and rephrase a little, as the instruction was confusing to me without having looked at the readme first:

```suggestion
2. Add tool details to the index in the root [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/README.md).
https://github.com/Azure/azure-sdk-tools/blob/main/README.md#index
```

- Pipelines should use common templates from `eng/pipelines/templates`.

> <<<<<<< HEAD

Reviewer: Bad merge here.
